test/unit/specs/logging.js

```diff
@@ -235,6 +233,34 @@ describe('<Logging>', function() {
     await check('trace')
   })
 
+  it('converts non-string values to strings', async () => {
+    const req = prepareMockRequest({})
+
+    logger.debug(0)
+    logger.debug(123)
+    logger.debug(true)
+    logger.debug(false)
+    logger.debug(null)
+    logger.debug(undefined)
+    logger.debug(_ => _)
+    logger.debug({foo: 'bar'})
+    logger.debug(['foo', 123, true, false, null, undefined, { bar: 123 }])
+
+    await Backendless.Logging.flush()
+
+    expect(req.body.map(b => b.message)).to.deep.equal([
+      '0',
+      '123',
+      "true",
+      "false",
+      "null",
+      undefined,
```

Comment:
I'm not sure `undefined` is a good idea, as a result we won't see any text in the log

what about converting it to:
`undefined` => `'undefined'`
`_=>_` => `[Function]`

what do you think?

Reply:
Could I clarify. If we try to log `4` and `"4"` we get identical messages?

Reply:
Yes, at the moment the result of entering `4` and `"4"` will be the same, I will add this to the tests and fix it for `undefined` and `function`

Reply:
@Valodya What if we add an `[<type>]` label for cases when we convert to string?

Reply:
hold on, 
- first of all, we have to keep the backward compatibility, so do not stringify a string
- for numbers/bool/etc do not need to add any additional prefix/suffix because the main idea was to avoid any issue when the value is not a string and simplify it by using in the Codeless 
- except `undefined` and `function` because it can not stringify it and returns `undefined` for both types